packages/plugins/plugin-markdown/src/components/Toolbar.stories.tsx

```diff
@@ -41,7 +42,7 @@ type StoryProps = {
 
 const DefaultStory = ({ content = '' }: StoryProps) => {
   const { themeMode } = useThemeContext();
-  const [text] = useState(DataType.makeText(content));
+  const [text] = useState(createObject(DataType.makeText(content)));
```

Comment:
(Can I get an explainer at some point about the difference between `createObject` and `Obj.make`?)